rfd/0147-repo-fork-syncing.md

```diff
@@ -0,0 +1,60 @@
+---
+authors: Mike Jensen (mike.jensen@goteleport.com)
+state: draft
+---
+ 
+# RFD 144 - Automatic Repo Syncing
+ 
+## What
+ 
+This RFD describes how we can sync a repo continuously and in an automatic way.  This can be used to provide private copies of a repo.
+ 
+## Why
+ 
+We currently face a few frictions which could be addressed through a more robust repo sync mechanism:
+1. Our internal `teleport-private` fork used for security development must be manually updated
+2. Some tooling (Dependabot) does not allow scanning for non-default branches, necessitating manual efforts to manage our release branches
```

Comment:
Honestly -- I might suggest we leave dependabot security alerts out of this RFD and handle those as a separate mini RFD.

Why? We'll need to sync forever, but we'll (hopefully) only need the dependabot hack for a little while until dependabot gets smarter.

Reply:
I don't think it makes sense as a separate RFD.  This is a single solution that will work for both needs.  I think highlighting how this solution can solve custom needs (like security scanning) helps illustrate how it's implemented.